contracts/pods/PodFactory.sol

```diff
@@ -0,0 +1,225 @@
+    }
+
+    /// @notice Get the signer threshold on the pod
+    /// @param podId Unique id for the orca pod
+    function getPodThreshold(uint256 podId)
+        external
+        view
+        override
+        returns (uint256)
+    {
+        address safe = getPodSafe(podId);
+        uint256 threshold = uint256(IGnosisSafe(safe).getThreshold());
+        return threshold;
+    }
+
+    /// @notice Get the next pod id
+    function getNextPodId() external view override returns (uint256) {
+        return memberToken.getNextAvailablePodId();
+    }
+
+    /// @notice Get the podAdmin from the base Orca controller
+    /// @dev Controller only allows existing admin to change
+    /// @param podId Unique id for the orca pod
+    function getPodAdmin(uint256 podId)
+        external
+        view
+        override
+        returns (address)
+    {
+        return IControllerV1(podController).podAdmin(podId);
+    }
+
+    //////////////////// STATE-CHANGING API ////////////////////
+
+    /// @notice Create a child Orca pod with optimistic timelock. Callable by the DAO and the Tribal Council
+    /// @param _members List of members to be added to the pod
+    /// @param _threshold Number of members that need to approve a transaction on the Gnosis safe
+    /// @param _podLabel Metadata, Human readable label for the pod
+    /// @param _ensString Metadata, ENS name of the pod
+    /// @param _imageUrl Metadata, URL to a image to represent the pod in frontends
+    /// @param _minDelay Delay on the timelock
+    /// @param _podAdmin The admin of the pod - able to add and remove pod members
+    function createChildOptimisticPod(
+        address[] calldata _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string calldata _ensString,
+        string calldata _imageUrl,
+        uint256 _minDelay,
+        address _podAdmin
+    ) external override onlyOwner returns (uint256, address) {
+        uint256 podId = memberToken.getNextAvailablePodId();
+
+        address safeAddress = createPod(
+            _members,
+            _threshold,
+            _podLabel,
+            _ensString,
+            _imageUrl,
+            _podAdmin,
+            podId
+        );
+
+        address timelock = createOptimisticTimelock(
+            safeAddress,
+            _minDelay,
+            podExecutor
+        );
+
+        // Set mapping from podId to timelock for reference
+        getPodTimelock[podId] = timelock;
+        getPodId[timelock] = podId;
+        latestPodId = podId;
+
+        emit CreatePod(podId, safeAddress);
+        return (podId, timelock);
+    }
+
+    /// @notice Create an Orca pod - a Gnosis Safe with a membership wrapper
+    function createPod(
+        address[] calldata _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string calldata _ensString,
+        string calldata _imageUrl,
+        address _podAdmin,
+        uint256 podId
+    ) internal returns (address) {
```

Comment:
should any of these be standardized? For example we own tribedao and feiprotocol.eth and could use those in an automated way for ens?

Reply:
Probably fine but just an idea

Reply:
Yep! Want to add standardised ENS names in a follow up PR that adjusts the deployment names. 

Maybe all pods are subdomains of the `tribedao` ENS name. `ops.tribedao.eth`, `fuse.tribedao.eth`, `feitribedao.eth` etc

Reply:
Update from Orca call: Orca doesn't currently support subdomains. Need to pass individual ENS name for each pod